charters/IngestSupport/charter.md

```diff
@@ -0,0 +1,104 @@
+# Ingest Support
+
+## Description
+
+The Ingest Support team is the liaison between the Data Coordination Platform (DCP) and labs contributing data to the Human Cell Atlas (HCA). This team closely collaborates with each data contributor to:
+
+*	Working with data contributors who are generating new data types to develop a deep understanding of their data
+* Assisting data contributors to describe their experiments in a manner that supports finability, reusability and reproducability with in the DCP ecosystem
+* Helping data contributors to submit their data to the DCP, resolving any emerging issues during the process
```

Comment:
The -ing should be removed from the first word of each bullet point so the combined sentences read:
"...collaborates with each data contributor to _work_ with..."
"...collaborates with each data contributor to _assist_ data..."
"...collaborates with each data contributor to _help_ data..."

Reply:
Thes statements are intentionally using "ing" to be in the active voice